core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -429,12 +428,12 @@ object ConsumerGroupCommand extends Logging {
     }
 
     private def getPartitionOffsets(group: String,
-                                    topicPartitions: Seq[TopicAndPartition],
+                                    topicPartitions: Seq[TopicPartition],
                                     channelSocketTimeoutMs: Int,
-                                    channelRetryBackoffMs: Int): Map[TopicAndPartition, Long] = {
-      val offsetMap = mutable.Map[TopicAndPartition, Long]()
+                                    channelRetryBackoffMs: Int): Map[TopicPartition, Long] = {
+      val offsetMap = mutable.Map[TopicPartition, Long]()
```

Comment:
Since this function really wants to work with `TopicAndPartition`, maybe we could save the conversion for when we return?